internal/virtualcircuit/create.go

```diff
@@ -0,0 +1,172 @@
+			}
+
+			data := make([][]string, 1)
+
+			if vrfFlag != "" {
+				data[0] = []string{vc.VrfVirtualCircuit.GetId(), vc.VrfVirtualCircuit.GetName(), string(vc.VrfVirtualCircuit.GetType()), strconv.Itoa(int(vc.VrfVirtualCircuit.GetSpeed())), vc.VrfVirtualCircuit.GetCreatedAt().String()}
+			} else {
+				data[0] = []string{vc.VlanVirtualCircuit.GetId(), vc.VlanVirtualCircuit.GetName(), string(vc.VlanVirtualCircuit.GetType()), strconv.Itoa(int(vc.VlanVirtualCircuit.GetSpeed())), vc.VlanVirtualCircuit.GetCreatedAt().String()}
+			}
+
+			header := []string{"ID", "Name", "Type", "Speed", "Created"}
+
+			return c.Out.Output(vc.VlanVirtualCircuit, header, &data)
+		},
+	}
+	// Virtual Circuit Params
+	createVirtualCricuitCmd.Flags().StringVarP(&connectionID, "id", "i", "", "The UUID of the interconnection.")
+	createVirtualCricuitCmd.Flags().StringVarP(&portID, "portId", "p", "", "The UUID of the port.")
+
+	// Common params for both VlanVirtualCircuit and VrfVirtualCircit
+	createVirtualCricuitCmd.Flags().IntVarP(&nnVlan, "vlan", "V", 0, "Adds or updates vlan  Must be between 2 and 4094")
+	createVirtualCricuitCmd.Flags().IntVarP(&speed, "speed", "s", 0, "bps speed or string (e.g. 52 - '52m' or '100g' or '4 gbps')")
+	createVirtualCricuitCmd.Flags().StringSliceVarP(&tags, "tags", "t", []string{}, "Return only the specified tags.")
+	createVirtualCricuitCmd.Flags().StringVarP(&name, "name", "n", "", "Name of the Virtual Circuit")
+	createVirtualCricuitCmd.Flags().StringVarP(&description, "description", "d", "", "Description for a Virtual Circuit")
+	createVirtualCricuitCmd.Flags().StringVarP(&projectID, "projectId", "P", "", "The UUID of the projectID.")
+
+	//VlanVirtualCircuit input params
+	createVirtualCricuitCmd.Flags().StringVarP(&vnid, "vnid", "v", "", "The UUID  of the VLAN.")
+
+	// VrfVirtualCircuit input params
+	createVirtualCricuitCmd.Flags().StringVarP(&md5, "md5", "M", "", "The plaintext BGP peering password shared by neighbors as an MD5 checksum")
+	createVirtualCricuitCmd.Flags().IntVarP(&peerAsn, "peerAsn", "a", 0, "The peer ASN that will be used with the VRF on the Virtual Circuit.")
+	createVirtualCricuitCmd.Flags().StringVarP(&vrf, "vrf", "R", "", "The UUID of the VRF that will be associated with the Virtual Circuit.")
+	createVirtualCricuitCmd.Flags().StringVarP(&subnet, "subnet", "S", "", "The /30 or /31 subnet of one of the VRF IP Blocks that will be used with the VRF for the Virtual Circuit. ")
+	createVirtualCricuitCmd.Flags().StringVarP(&customerIP, "customerIP", "c", "", "An IP address from the subnet that will be used on the Customer side")
+	createVirtualCricuitCmd.Flags().StringVarP(&metalIP, "metalIP", "m", "", "An IP address from the subnet that will be used on the Metal side. ")
+
+	_ = createVirtualCricuitCmd.MarkFlagRequired("id")
+	_ = createVirtualCricuitCmd.MarkFlagRequired("portId")
+	_ = createVirtualCricuitCmd.MarkFlagRequired("projectId")
+	return createVirtualCricuitCmd
```

Comment:
Nit: this variable should be `createVirtualCircuitCmd` (i.e. `Circuit` instead of `Cricuit`).